image/v3/modifying/text-fonts.md

```diff
@@ -135,13 +135,13 @@ Rotate the text block clockwise with a desired angle.
 
 ### Line height
 
-> public FontInterface::lineHeight(float $height): FontInterface
+> public FontInterface::setLineHeight(float $height): FontInterface
```

Comment:
Please revert. This is correct call of class Intervention\Image\Typography\FontFactory::class.